packages/@magic-sdk/provider/src/modules/auth.ts

```diff
@@ -53,9 +49,9 @@ export class AuthModule extends BaseModule {
     const { email } = configuration;
     const requestPayload = createJsonRpcRequestPayload(
       this.sdk.testMode ? MagicPayloadMethod.LoginWithEmailOTPTestMode : MagicPayloadMethod.LoginWithEmailOTP,
-      [{ email, showUI: true }],
```

Comment:
With this change, EmailOTP can hide the pop window now. Am I correct?

Reply:
That is correct. Good catch @Ethella